src/main/java/com/minecolonies/apiimp/initializer/ModBuildingsInitializer.java

```diff
@@ -437,6 +441,13 @@ public static void init(final RegistryEvent.Register<BuildingEntry> event)
                                    .addBuildingModuleProducer(() -> new SettingsModule()
                                                                       .with(AbstractBuildingWorker.BREEDING, new BoolSetting(true))
                                                                       .with(BuildingBeekeeper.MODE, new StringSetting(BuildingBeekeeper.HONEYCOMB, BuildingBeekeeper.HONEY, BuildingBeekeeper.BOTH)), SettingsModuleView::new)
+                                   .addBuildingModuleProducer(() -> new ItemListModule(BUILDING_FLOWER_LIST),  () -> new ItemListModuleView(BUILDING_FLOWER_LIST, COM_MINECOLONIES_COREMOD_REQUEST_FLOWERS, false,
+                                     (buildingView) ->
+                                   {
+                                       Set<ItemStorage> flowers = new HashSet<>();
+                                       ItemTags.FLOWERS.getAllElements().forEach((item) -> flowers.add(new ItemStorage(new ItemStack(item))));
```

Comment:
maybe hide this in a utility method somewhere.

Reply:
Could be a static method in the CompatibilityManager, if the cook one is similar then yes, please

Reply:
OK, I'll move those two then.

Reply:
Done.